deployment/clouddeploy/osv-api/run-batch.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: osv-grpc-backend-batch
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: '100'
```

Comment:
should we bump this up a bit more?

Reply:
Currently, we hover around 100 instances serving all requests with 5 concurrency. Batch queries are < 5% of all our queries so I'd assume it'd use <25 instances (give or take).

I'll leave it at the default 100, but will keep an eye on it in case my dodgy maths it too dodgy.